addons/sys_core/fnc_processRadioSpeaker.sqf

```diff
@@ -25,14 +25,12 @@ private ["_okRadios", "_functionName"];
 params ["_unit","_playerRadios"];
 TRACE_2("",_unit,_playerRadios);
 
-private _radioId = _unit getVariable [QGVAR(currentSpeakingRadio), ""];
+private _radioId = _unit getVariable QGVAR(currentSpeakingRadio);
```

Comment:
Is a default no longer necessary? Why checking then on line 29 `if (_radioId == "") exitWith { false };` and later in line 33 `if (isNil "_radioId") exitWith { false };`

Reply:
Default value was added here https://github.com/IDI-Systems/acre2/commit/28cd75cd9ba176742a1c304423dad07b3fa6e3c3#diff-bebd4cbc667fc63e690cc00cdee69f6a, because that was my first hunch where the problem is. So you are correct, we can remove it.